src/helpers/config.ts

```diff
@@ -25,6 +25,19 @@ export const DEFAULT_USERNOTE_TYPES: readonly RawUsernoteType[] = [
 	{key: 'botban', color: 'black', text: 'Bot Ban'},
 ];
 
+/** Default subreddit configuration to use if subreddit doesnt have one.
+ *  Empty strings are used in the default config by the plugin even though
+ *  this doesn't match the type exactly.
+ */
```

Comment:
Whitespace nit for consistency:

```suggestion
/**
 * Default subreddit configuration to use if subreddit doesnt have one.
 * Empty strings are used in the default config by the plugin even though
 * this doesn't match the type exactly.
 */
```